codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/rustlang/RustWriter.kt

```diff
@@ -624,7 +624,13 @@ class RustWriter private constructor(
         init {
             expressionStart = '#'
             if (filename.endsWith(".rs")) {
-                require(namespace.startsWith("crate") || filename.startsWith("tests/") || filename == "build.rs") {
+                require(
+                    namespace.startsWith("crate") ||
+                    // Unix systems
+                    filename.startsWith("tests/") ||
```

Comment:
Could you not use `File.separator` and collpase unix/windows branch?

Reply:
TIL about `File.separator`. TY Aaron.